### PR TITLE
api-endpoints

### PR DESCRIPTION
**Pull request by yannsf29**

### CODE REVIEW

File: app/controllers/cats_controller.rb [hunk 1]
```diff
@@ -1,2 +1,35 @@
...
+     
+    
+    
+    # Handle strong parameters, so we are secure 
+     private
+ def cat_params
+ params.require(:cat).permit(:name, :age, :enjoys, :image)
+    
+
+   end
```
Comment (WalkerTexas-coder) [inline app/controllers/cats_controller.rb]: Please clean up the white space and indentation.

**Review by WalkerTexas-coder — CHANGES_REQUESTED** [review]
One change requested

**Comment by Echo5Mike** [comment]
Cleaned up code.

**Review by WalkerTexas-coder — APPROVED** [review]
Okay let's keep it going.

**Pull request merged**